meson_options.txt

```diff
@@ -8,9 +8,9 @@ option('introspection', type: 'feature',
        value: 'auto',
        yield: true,
        description: 'Enable GObject Introspection (depends on GObject)')
-option('gcc_vector', type: 'boolean',
+option('vector_intrinsics', type: 'boolean',
```

Comment:
You should not rename the configuration option: this will break builds—just like it broke CI.

One option is to add a `vector_intrinsics` option and deprecate `gcc_vector`, but it requires bumping up the version of Meson to 0.60: https://mesonbuild.com/Build-options.html#deprecated-options

Reply:
sure, sounds good